### DISCUSSION TITLE
What's DFV stand for?

### DISCUSSION THREAD

**Discussion by datgausaigon** [pods-framework/pods#6902] [Q&A] [answered]
Hi, 
What's DFV in Pods DFV stand for? Data Form View?
Thanks

**Comment by sc0ttkclark** [comment, marked as answer]
Hey there 👋

DFV stands for Dynamic Field Views.

When Pods first started, all of the form inputs would be output using HTML rendered through PHP. Then all of the custom JS code had to run separate from the HTML.

Then we created the DFV code that is told through JSON on the page how a specific field is supposed to function. The newest iteration of DFV is powered by React.

Without DFV, we could not do repeatable fields and other new features we are working on next :)

**Reply by datgausaigon** [reply]
❤️❤️❤️
Thank you
